polkadot/node/network/statement-distribution/src/v2/grid.rs

```diff
@@ -908,6 +913,14 @@ struct MutualKnowledge {
 	/// `Some` only if we have advertised, acknowledged, or requested the candidate
 	/// from them.
 	local_knowledge: Option<StatementFilter>,
+	/// Knowledge peer sent to us, this is different from
+	/// `local_knowledge` and `remote_knowledge`, through the fact that includes only
```

Comment:
I think this is not clear enough. Just by reading these comments, `remote_knowledge` gets updated when remote peers have advertised, acknowledged, or requested the candidate, but  `received_knowledge` also does the same as per comment.

Reply:
I'll fix the comment.